.eslintrc.json

```diff
@@ -0,0 +1,69 @@
+{
+  "env": {
+    "browser": true,
+    "jquery": true,
+    "es2021": true
+  },
+  "extends": [
+    "semistandard"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 12
+  },
+  "rules": {
+    "max-len": [
+      "warn",
+      {
+        "code": 120
+      }
+    ],
```

Comment:
The extra rules below this line are the ones our code still violates, and which would need to be manually fixed.